src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -203,7 +207,7 @@ export class NotebookMarkdownRenderer {
 			href = this.resolveUrl(base, href);
 		}
 		try {
-			href = encodeURI(href).replace(/%5C/g, '\\').replace(/%25/g, '%');
+			href = encodeURI(href).replace(/%5C/g, '\\').replace(/%25/g, '%').replace(/%7C/g, '|');
```

Comment:
The | unescape needs to happen first, otherwise unescaping the % could result in things mistakenly being unescaped into a |.

Reply:
so this is the correct way ?-> encodeURI(href).replace(/%5C/g, '\\').replace(/%7C/g, '|').replace(/%25/g, '%');

Reply:
Yes, that should be fine.